src/unit.rs

```diff
@@ -1529,6 +1529,17 @@ impl<'input, Endian> Attribute<'input, Endian>
             _ => return None,
         })
     }
+
+    /// Try to convert this attribute's value to a string.
```

Comment:
I slightly reworded the short description, and used your text as the body. I wanted to keep the short description similar to the other functions.